apps/charge-accounts-service/src/operators/operators.service.ts

```diff
@@ -67,12 +67,15 @@ export class OperatorsService {
   }
 
   async createWallet (createOperatorWalletDto: CreateOperatorWalletDto): Promise<OperatorWallet> {
-    const createdOperatorWallet = new this.operatorWalletModel(createOperatorWalletDto)
+    const createdOperatorWallet = new this.operatorWalletModel({
+      ...createOperatorWalletDto,
+      smartWalletAddress: createOperatorWalletDto.smartWalletAddress.toLowerCase()
+    })
     return createdOperatorWallet.save()
   }
 
   async findWallet (key: string, value: string): Promise<OperatorWallet> {
-    return this.operatorWalletModel.findOne({ [key]: value })
+    return this.operatorWalletModel.findOne({ [key]: value.toLowerCase() })
```

Comment:
@LiorAgnin we're using [findWallet for ownerId](https://github.com/fuseio/fusebox-backend/blob/9a672ff33b5ee20d62380b5b93853818aa07706b/apps/charge-accounts-service/src/operators/operators.controller.ts#L73C60-L73C67) as well, maybe need to add a new method.